src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,627 @@
+# Dynamic proxy configuration
+
+The dynamic proxy configuration feature allows operations to provide a customized proxy configuration that would be updated automatically according to the applications that are running within ConductR cluster.
+
+ConductR provides a default configuration with can be overridden should the need arise.  For example, HAProxy can provide HTTP Basic Auth for staging deployments enabling protected preview prior to promoting to production. This is accomplished by deploying a template haproxy.cfg file as part of the configuration of the HAproxy bundle.
+
+This document is a guide and reference for using dynamic HAProxy configuration templates.
+
+
+## Assumptions
+
+ConductR uses [HAProxy](http://www.haproxy.org/) as its proxying solution. It is assumed the reader has some familiarity with HAProxy installation and configuration.
+
+ConductR makes use of [Handlebars](http://handlebarsjs.com/) as the templating solution to generate the customised HAProxy configuration. It is assumed the reader has some degree familiarity in working with other templating solutions.
+
+## Requirement
+
+ConductR makes the assumption that an endpoint must be uniquely identified by its name given a particular system and system version.
+
+The endpoints that failed to satisfy this requirement will be considered as duplicate, and the duplication is assumed to be a result of configuration error. Because of this, duplicated endpoints will not be exposed via proxy configuration.
+
+## Creating and deploying custom HAProxy configuration
+
+It is possible to provide custom HAProxy configuration by supplying configuration overrides to the ConductR HAProxy bundle.
+
+[ConductR CLI](CLI) is required to perform the following steps, and as such it needs to be installed beforehand.
+
+These are the steps to deploying custom HAProxy configuration.
+
+### Setup
+
+Create a directory where the custom HAProxy configuration will be placed, e.g.
+
+```
+mkdir -p /tmp/custom-haproxy-conf
```

Comment:
"Solved by PR". Actually, my PR uses now `lang-none`. Terminal commands shouldn't be highlighted because there is no terminal highlight support by [code-prettify](https://github.com/google/code-prettify). The tag `lang-none` doesn't work yet, but there is at least a PR pending: https://github.com/google/code-prettify/pull/396